hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -2792,6 +2792,22 @@ void onFinished() {
     @Override
     void onError(Throwable error) {
       LOG.info("{} failed with {}", getDescription(), error.getMessage());
+<<<<<<< HEAD
+    }
+  }
+
+  private static class RestoreBackupSystemTableProcedureBiConsumer extends ProcedureBiConsumer {
+
+    @Override
+    void onFinished() {
+      LOG.info("RestoreBackupSystemTableProcedure completed");
+    }
+
+    @Override
+    void onError(Throwable error) {
+      LOG.info("RestoreBackupSystemTableProcedure failed with {}", error.getMessage());
+=======
+>>>>>>> 1e06bccf79ede58e86ca7f1f4b9231bb7a53fd72
```

Comment:
bad merge

Reply:
ah thank you, don't know how I missed this